knexfile.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
 
   test: {
     client: 'pg',
-    connection: 'postgres://localhost/school_test',
+    connection: 'postgres://localhost/',
     migrations: {
```

Comment:
I wonder if we need to incorporate a `testing environment` when testing the Express calls but I am unsure how will this look like. Now that I think of it, this might be out of the scope for this project tho.

Reply:
From the testing lesson it didn't seem like it was much different than just making another database and running commands a little differently.